pkg/linear-pools/contracts/tetu-linear-pool/TetuLinearPool.sol

```diff
@@ -24,11 +24,8 @@ import "./interfaces/ITetuSmartVault.sol";
 import "./TetuShareValueHelper.sol";
 
 contract TetuLinearPool is LinearPool, Version, TetuShareValueHelper {
-    IERC20 private immutable _mainToken;
```

Comment:
_wrappedToken is a private variable inside LinearPool.sol, that's why this variable was recreated. I removed _mainToken, since it's not used

Reply:
We don't have direct access to `_wrappedToken`, but we can utilize the `getWrappedToken()` [getter](https://github.com/balancer/balancer-v2-monorepo/blob/master/pkg/pool-linear/contracts/LinearPool.sol#L208) instead. We should do this instead of storing a redundant variable.